slickml/plotting.py

```diff
@@ -805,3 +810,176 @@ def plot_shap_summary(
         color_bar_label=color_bar_label,
     )
     plt.show()
+
+
+def plot_regression_metrics(figsize=None, **kwargs):
+    """Function to plot regression metrics.
+    This function is a helper function based on the plotting_dict
+    attribute of the RegressionMetrics class.
+    Parameters
+    ----------
+    figsize: tuple, optional, (default=(12, 12))
+        Figure size
+    """
+
+    # initializing figsize
+    if figsize is None:
+        figsize = (12, 16)
+    elif isinstance(figsize, list) or isinstance(figsize, tuple):
+        figsize = figsize
+    else:
+        raise TypeError("Only tuple and list types are allowed for figsize.")
+
+    fig, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(3, 2, figsize=figsize)
```

Comment:
From the README output, we might want to add a small spacing between the subplots (the titles and x-axis label almost overlap). The following could be added for both regression and classification with wspace and hspacing being tunable parameters. 

plt.subplots_adjust(wspace=None, hspace=0.20)

Reply:
yeah. I thought about it. @eitanlees  would take care of the plotting/formatting soon when he finds some time.